packages/tools/src/tools/CrosshairsTool.ts

```diff
@@ -76,7 +76,7 @@ interface CrosshairsAnnotation extends Annotation {
 }
 
 function defaultReferenceLineColor() {
-  return 'rgb(200, 200, 200)';
+  return 'rgb(0, 200, 0)';
```

Comment:
Do we really need this funciton at all? It's weird that it defines its own color seperate from everything else. Shouldn't the default just be one of the colors from the default tool style? I.e. `colorHighlighted`? Then we could just remove this function entirely and nuke this isolated state.

Reply:
That is more work to do I will do it in another PR